src/main/scala/nl.knaw.dans.easy.stage/dataset/EMD.scala

```diff
@@ -52,6 +58,33 @@ object EMD extends DebugEnhancedLogging {
     }
   }
 
+  def addAgreementFields(emd: EasyMetadata)(implicit s: Settings): Unit = {
+    val agreementPath = s.bagitDir.toPath.resolve(Paths.get("metadata/agreements.xml"))
+    if (Files.exists(agreementPath)) {
+      val agreementsXml = XML.loadFile(agreementPath.toFile)
+      if (BooleanUtils.toBoolean((agreementsXml \\ "depositAgreementAccepted").text)) {
+        emd.getEmdRights.setAcceptedLicense(true)
```

Comment:
Have you tested that EMD can have _**both**_ a `<dct:license>accepted<dct:license>` and another license as chosen by `easy-deposit-ui`?

Reply:
<emd:easymetadata xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dct="http://purl.org/dc/terms/" xmlns:eas="http://easy.dans.knaw.nl/easy/easymetadata/eas/" xmlns:emd="http://easy.dans.knaw.nl/easy/easymetadata/" emd:version="0.1">
  <emd:title>
    <dc:title xml:lang="nld">DOOM 2 message from depositor</dc:title>
  </emd:title>
  <emd:creator>
    <eas:creator>
      <eas:initials>V.</eas:initials>
      <eas:surname>Putin</eas:surname>
    </eas:creator>
  </emd:creator>
  <emd:description>
    <dc:description xml:lang="nld">jahaaaaaaaaa</dc:description>
  </emd:description>
  <emd:contributor>
    <eas:contributor>
      <eas:initials>V.</eas:initials>
      <eas:surname>Putin</eas:surname>
      <eas:organization>Braaap</eas:organization>
      <eas:role eas:scheme="DATACITE">RightsHolder</eas:role>
    </eas:contributor>
  </emd:contributor>
  <emd:date>
    <eas:created eas:format="MILLISECOND" eas:scheme="W3CDTF">2019-04-28T00:00:00.000+02:00</eas:created>
    <eas:available eas:format="MILLISECOND" eas:scheme="W3CDTF">2019-05-02T13:51:12.000+02:00</eas:available>
    <eas:dateSubmitted eas:format="DAY" eas:scheme="W3CDTF">2019-05-02T00:00:00.000+02:00</eas:dateSubmitted>
  </emd:date>
  <emd:identifier>
    <dc:identifier eas:identification-system="http://www.persistent-identifier.nl" eas:scheme="PID">urn:nbn:nl:ui:13-gr-g07s</dc:identifier>
    <dc:identifier eas:identification-system="https://doi.org" eas:scheme="DOI">10.5072/dans-ibpE-zm3-7dsc</dc:identifier>
    <dc:identifier eas:scheme="DMO_ID">easy-dataset:41</dc:identifier>
  </emd:identifier>
  <emd:rights>
    <dct:accessRights eas:schemeId="common.dcterms.accessrights">OPEN_ACCESS</dct:accessRights>
    <dct:license eas:scheme="Easy2 version 1">accept</dct:license>
    <dct:rightsHolder>V. Putin (Braaap)</dct:rightsHolder>
  </emd:rights>
  <emd:audience>
    <dct:audience eas:schemeId="custom.disciplines">easy-discipline:60</dct:audience>
  </emd:audience>
  <emd:other>
    <eas:remark>containsPrivacySensitiveData</eas:remark>
    <eas:remark>Hi!</eas:remark>
    <eas:application-specific>
      <eas:metadataformat>ANY_DISCIPLINE</eas:metadataformat>
      <eas:pakbon-status>NOT_IMPORTED</eas:pakbon-status>
      <eas:archive eas:location="DATAVAULT"></eas:archive>
    </eas:application-specific>
    <eas:etc></eas:etc>
  </emd:other>
</emd:easymetadata>

Reply:
I think so

![image](https://user-images.githubusercontent.com/24325101/57073780-700fc080-6ce2-11e9-87ed-4ace47e54c66.png)

Reply:
I don't think so 😉
Where is the 'actual' license you selected to go with the `OPEN_ACCESS` `dct:accessRights`? Like `CC-BY-NC-SA-4.0`? I don't see it in the EMD above... Is it not converted from DDM to EMD?

Reply:
I guess not, i will check what the master inserts if i do the same thing.

Reply:
i have frozen easy-ingest-flow to see the input of a typical depoist. I get this a XML:

![image](https://user-images.githubusercontent.com/24325101/57139322-45426c80-6db5-11e9-9a75-5ba4b96f46e1.png)
I think we should add more to the agreements.xml to add the license to the emd? Or is this saved in an uri or other xml file? Perhaps we should discuss this next week.

Reply:
If we add the uri field in the deposit-api =>  `xsi:type="dcterms:URI"` the license will be translated to the EMD. 

<img width="638" alt="Screenshot 2019-05-07 at 15 44 28" src="https://user-images.githubusercontent.com/24325101/57304338-6f5e9c00-70df-11e9-9262-b6190c23a222.png">

Reply:
I sent an email to @lindareijnhoudt about this issue with `easy-deposit-api`. I think with that, this thread is resolved.